### PR TITLE
CB-10980 - Validate FreeIPA backup permission after after deployment

### PR DESCRIPTION
**Pull request by jamisonbennett**
Before installing FreeIPA, perform a test to ensure the instance has
write access to the object store for backup files. This improves the
delay before failues are reported. Previously it would take more than
40 minutes, now an error is reported within 5 minutes.

This was validated by timing how long it took to fail when deployed
using a local instance of cloudbreak and deployming on AWS.

See detailed description in the commit message.

### CODE REVIEW

**Review by oleewere — APPROVED** [review]

**Review by lacikaaa — APPROVED** [review]
LGTM! thanks for the changes

**Comment by oleewere** [comment]
please rebase the branch

**Comment by jamisonbennett** [comment]
this was also reviewed https://github.com/hortonworks/cloudbreak/pull/9922 for master.

File: freeipa/src/main/resources/freeipa-salt/salt/freeipa/backups.sls [hunk 1]
```diff
@@ -1,25 +1,5 @@
...
-
-/usr/local/bin/backup-log-filter.sh:
-  file.managed:
-    - makedirs: True
-    - user: root
-    - group: root
-    - mode: 700
-    - source: salt://freeipa/scripts/backup-log-filter.sh
+include:
+  - freeipa.backup-common
```
Comment (christmasferret) [inline freeipa/src/main/resources/freeipa-salt/salt/freeipa/backups.sls:2]: Good refactoring.

**Pull request merged**